docs/introduction/getting-started.story.mdx

```diff
@@ -37,34 +37,38 @@ This will create the folder `my-app` and initialize a new project inside.
 Already have a project? You can install Circuit UI via `yarn` or `npm`. In your current project directory, run the following command:
 
 ```bash
+# With yarn:
 yarn add @sumup/circuit-ui
-# Or if you prefer npm:
+# With npm:
 npm install @sumup/circuit-ui
 ```
 
-Circuit UI relies on some mandatory peer dependencies, namely [React](https://reactjs.org/) and [Emotion](https://github.com/emotion-js/emotion), our CSS-in-JS library. You should install them with the following command:
+Circuit UI relies on some mandatory peer dependencies, namely [@sumup/icons](https://www.npmjs.com/package/@sumup/icons), [React](https://reactjs.org/), and [Emotion](https://emotion.sh/). You should install them with the following command:
 
 ```bash
-yarn add react react-dom @emotion/core @emotion/styled emotion-theming
-# Or if you prefer npm:
-npm install react react-dom @emotion/core @emotion/styled emotion-theming
+# With yarn:
+yarn add @sumup/icons react react-dom @emotion/core @emotion/styled emotion-theming
```

Comment:
In a typical project, wouldn't react and react-dom already be installed? And if they're not, shouldn't we recommend starting with create-sumup-react-app?

Reply:
We already recommend `create-sumup-react-app` in the section above this one :) 

I'm working on upgrading `create-sumup-react-app` and `create-sumup-next-app` alongside Circuit UI v2 and I will update the getting started docs once all the breaking changes have been merged.